apps/backend/src/modules/quiz-category/quiz-category.entity.ts

```diff
@@ -0,0 +1,69 @@
+import { type Entity } from "~/libs/types/types.js";
+
+type QuizCategoryObject = {
+	createdAt: string;
+	id: number;
+	name: string;
+	updatedAt: string;
+};
+
+type NewQuizCategoryObject = {
+	createdAt: string;
+	name: string;
+	updatedAt: string;
+};
```

Comment:
No need to have these types

Reply:
About the category entity and services from PR #208, do I need to merge it with my category services or keep it separate?

Reply:
I have basic implementation of categories module in my PR #208 including all methods in service and repository, but I mostly needed this module for scores from joined table with users. Scores logic is really important there. However, my new task also depends on categories module ( I need to get all scores, new endpoint for that). So if it would be really great if you’ll use #208  implementation and change all logic that you need (like findAll) to match your implementation.

Reply:
I want to rename your category type into `CategoryWithScores` since my implementation didn't require any scores, but it will make conflicts with your future changes to your other tasks. I may keep the types separate but will still try to merge our services for now.

![Screenshot (338)](https://github.com/user-attachments/assets/fae18160-33d8-4580-9f01-bc71332dd227)